bindings/tpython/src/TPython.cxx

```diff
@@ -143,6 +143,12 @@ Bool_t TPython::Initialize()
       // force loading of the ROOT module
       PyRun_SimpleString(const_cast<char *>("import ROOT"));
    }
+   
+   // verify that import ROOT was successful (and of its depending cppyy lib)   
```

Comment:
May I ask a couple of things:
1. Could you do this check inside ` if (!Py_IsInitialized()) {` and, instead of checking `gThisModule`, check directly the return value of `PyRun_SimpleString` to know if import ROOT failed?

https://docs.python.org/3/c-api/veryhigh.html#c.PyRun_SimpleStringFlags

2. Please combine the three commits which are not typo fixing into one.

Thanks again!

Reply:
Changes done in https://github.com/root-project/root/pull/8043